src/ngraph/op/add.hpp

```diff
@@ -30,8 +30,8 @@ namespace ngraph
         {
         public:
             NGRAPH_API
-            static const std::string type_name;
-            const std::string& description() const override { return type_name; }
+            static constexpr NodeTypeInfo type_info{"Add", 0};
+            const NodeTypeInfo& get_type_info() const override { return type_info; }
             /// \brief Constructs an unitialized addition operation
```

Comment:
Not part of your PR, but probably means to say "uninitialized" addition operation?